ParserGeneric/tti-command_param_data_type.txt

```diff
@@ -365,7 +365,7 @@ minimumValue: number
 maximumValue: number
 
 display.input.option()
-n: display_BUTTON_OPTIONs | nil 
+BUTTON_OPTIONn: display_BUTTON_OPTIONs | nil 
```

Comment:
Is the `n` intended here?

Reply:
This is return type from display.input.option() , which is custom created Enum constants.